pyfr/plugins/fluidforce.py

```diff
@@ -162,6 +137,7 @@ def __call__(self, intg):
 
         # Solution matrices indexed by element type
         solns = dict(zip(intg.system.ele_types, intg.soln))
+        cgrads = dict(zip(intg.system.ele_types, intg.grad_soln))
```

Comment:
I think we want this in an if statement otherwise the plugin will no longer work for Euler.